frontstage/views/account/account_transfer_survey.py

```diff
@@ -188,26 +149,23 @@ def is_max_transfer_survey_exceeded(selected_businesses, form):
 @account_bp.route("/transfer-surveys/survey-selection", methods=["POST"])
 @jwt_authorization(request)
 def transfer_survey_post_survey_select(_):
```

Comment:
This needs refining, I struggle to understand what is going on here, the surveys_selected seems to be passed around a number of times, called again, then flask session is used to work out validation.

Reply:
I've delved a bit deeper into it and have removed redunent processing of flask_sessions